lib/common/common/src/bitvec.rs

```diff
@@ -0,0 +1,11 @@
+use bitvec::slice::BitSlice as RawBitSlice;
+pub use bitvec::store::BitStore;
+pub use bitvec::vec::BitVec as RawBitVec;
+
+// BitVec type we use across the codebase. Has consistent bit layout, suitable for persisting.
+pub type BitSliceElement = u64;
+pub type BitSlice = RawBitSlice<BitSliceElement, bitvec::order::Lsb0>;
+pub type BitVec = RawBitVec<BitSliceElement, bitvec::order::Lsb0>;
+
+// BitVec type we use for in-memory bit vectors. Can be more efficient than BitVec.
+pub type MemoryBitVec = bitvec::vec::BitVec;
```

Comment:
Explicitly specify container type and bit order of `BitVec` and `BitSlice` types in a central place here.

Reply:
An idea: It would be helpful if the type system would prevent converting `&[u8]`/mmaps into this type and vice versa.